libraries/common/components/SurroundPortals/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Portal from '../Portal';
+import { AFTER, BEFORE } from '../../constants/Portals';
+
+/**
+ * The Portal component.
+ * @param {string} base portal (center) name
+ * @param {Object} [portalProps={}] for portals
+ * @param {Function} [children=null] children
+ * @returns {JSX}
+ */
+const SurroundPortals = ({ portalName, portalProps = {}, children }) => (
```

Comment:
I don't like this component. We lose control over the portals in this way. This just adds a new layer of complexity to the already complex portals system.

Reply:
In my opinion it simplifies the usage of portals without making it less flexible, because it's optional. This reduces the overhead for creating, maintaining and reviewing portals and increases testability.